code/game/objects/items/stunbaton.dm

```diff
@@ -1,34 +1,50 @@
 /obj/item/melee/baton
 	name = "stun baton"
 	desc = "A stun baton for incapacitating people with."
+
 	icon_state = "stunbaton"
 	item_state = "baton"
 	lefthand_file = 'icons/mob/inhands/equipment/security_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/security_righthand.dmi'
-	slot_flags = ITEM_SLOT_BELT
+
 	force = 10
-	throwforce = 7
-	w_class = WEIGHT_CLASS_NORMAL
 	attack_verb = list("beaten")
+
+	w_class = WEIGHT_CLASS_NORMAL
+	slot_flags = ITEM_SLOT_BELT
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 50, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 80)
 
-	var/cooldown_check = 0
+	throwforce = 7
+	var/throw_stun_chance = 35
+
+	var/attack_cooldown_check = (0 SECONDS)

```

Comment:
What's with all the ()'s, they're not necessary.

Reply:
following wrong precedent